sleap/gui/widgets/video.py

```diff
@@ -1568,6 +1569,9 @@ def mousePressEvent(self, event):
             # Shift-click to mark all points as complete
             elif event.modifiers() == Qt.ShiftModifier:
                 self.parentObject().updatePoints(complete=True, user_change=True)
+            # Ctrl-click to duplicate instance
+            elif event.modifiers() == Qt.ControlModifier:
+                self.parentObject().duplicate_instance()
```

Comment:
Let's defer this to the QtInstance.mousePressEvent where we also call QtInstance.duplicate_instance()